### ISSUE TITLE
Testing these models on Retinal Vessel Segmentation

### ISSUE THREAD

**Issue by DeVriesMatt** [LeeJunHyun/Image_Segmentation#61]
Hi, thanks for the brilliant repo.

I was wondering if you may have trained and tested these models on retinal segmentation tasks. I am currently doing so and the R2U-Net produces great training results however for validation and test sets, it returns all-black images. 

Do you have any suggestions to what may fix this?

Thanks again!

**Comment by sdc17** [comment]
Hey, I have the same problem with you when  I was training R2U-Net and R2AttU-Net on my own dataset. Have you solved this problem yet?  These two models produced good training scores but really bad test scores while AttU-Net worked well on both training and testing. Thanks!

**Comment by DeVriesMatt** [comment]
@sdc17 Yeah, I still haven't been able to figure out the problem. I have tried adding dropout, using data augmentation, and tried many different loss functions however nothing has worked. Please let me know if you figure anything out.

**Comment by xlh1135355585** [comment]
I also conducted experiments on this data set and found that the final experimental effects of these networks are almost the same. Have you encountered this problem?
